.golangci.yml

```diff
@@ -0,0 +1,9 @@
+issues:
+  exclude-rules:
+    - path: pkg/kubefed2/enable/enable.go
+      linters:
+        - unused
+        - structcheck
+    - path: pkg/kubefed2/disable.go
+      linters:
+        - structcheck
```

Comment:
(suggestion) I would wish there are no exceptions and all code follow similar pattern

Reply:
I came up with commit to remove the exceptions in golangci.yml, maybe you could take a look. https://github.com/kubernetes-sigs/federation-v2/pull/790/commits/9634a69c28cafaef091df727297d9b3e3d2c68e3